var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
Thats not _really_ a tarball. Its basically empty, a dummy that is there to satisfy spack's need to have something to install. I don't like the idea of hosting  some dummy tarball in some mirror solely to affect that, especially since the tarball is a dummy thing. There is no FASTMath package per-se. Its just an uber install convenience promised as part of FASTMath project. Does that make sense?